pybossa/cache/task_browse_helpers.py

```diff
@@ -246,8 +246,10 @@ def parse_tasks_browse_args(args):
         parsed_args['priority_from'] = float(args['priority_from'])
     if args.get('priority_to') is not None:
         parsed_args['priority_to'] = float(args['priority_to'])
-    if args.get('display_columns'):
-        parsed_args['display_columns'] = json.loads(args['display_columns'])
+        parse_args["display_columns"] = args['display_columns'] \
```

Comment:
think we need to fix the indentation, parsing "display_columns" should be independent from parsing "priority_to".